### ISSUE TITLE
How about date_format_* strings now?

### ISSUE THREAD

**Issue by ghost** [BurgerZ/WeatherBZ#41]
Should we translate or not? Example: In German it has to be <string name="date_format_dmy">dd.MM.yyyy</string> and not as per the source file <string name="date_format_dmy">dd/MM/yyyy</string>

**Comment by BurgerZ** [comment]
No strings - no problems!))
date_format_\* strings must be only in values folder (or in yours, but with similar text)

**Comment by vagyula1** [comment]
Is it possible to add date format to yyyy.MM.dd or MMMM dd.EEEE?

**Comment by BurgerZ** [comment]
@vagyula1 Yes, I'll add whatever you want) Just tell me what formats I need to add.

**Comment by vagyula1** [comment]
Great.
These formats would be needed:
Wednesday (only)
2013.11.06.
2013.nov.06.We
Nov.06.We
11.06.W

Thanks

**Comment by ghost** [comment]
Leave me out of your conversation, will you.

Mit besten Grüßen, 
Peter McBride

Sent from smartphone, therefore quick and dirty.

Am 06.11.2013 um 09:59 schrieb "vagyula1" notifications@github.com:

Great.
These formats would be needed:
Wednesday (only)
2013.11.06.
2013.nov.06.We
Nov.06.We
11.06.W

Thanks

**Comment by ghost** [comment]
is possible to add multilang support for the date here ? : 
should be 'VEN. 6 DEC." for my language
![screenshot_2013-12-06-07-58-27](https://f.cloud.github.com/assets/812995/1698081/4da2a1a6-5f24-11e3-8a00-8edb8864d8c1.png)

**Comment by BurgerZ** [comment]
@cpasmoi OK.

**Comment by ghost** [comment]
i just checked and widgets date format need to be translated also, same format as above, thanks
